packages/docs-app/src/examples/datetime-examples/dateTimePickerExample.tsx

```diff
@@ -16,23 +16,34 @@
 
 import * as React from "react";
 
-import { Classes } from "@blueprintjs/core";
+import { Classes, H5, Switch } from "@blueprintjs/core";
```

Comment:
changes to this file are unrelated to the issue, please revert them. also this is a deprecated component, people shouldn't be using this docs page anyway.

Reply:
Reverted.